### PR TITLE
Fix typos in syntax

### PR DESCRIPTION
**Pull request by tomjridge**

### CODE REVIEW

File: spec/13-syntax-summary.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ grammar:
   Expr1             ::=  `if' `(' Expr `)' {nl} Expr [[semi] `else' Expr]
                       |  `while' `(' Expr `)' {nl} Expr
                       |  `try' (`{' Block `}' | Expr) [`catch' `{' CaseClauses `}'] [`finally' Expr]
-                      |  `do' Expr [semi] `while' `(' Expr ')'
+                      |  `do' Expr [semi] `while' `(' Expr `)'
```
Comment (som-snytt) [inline spec/13-syntax-summary.md:135]: I wonder why the quoting style isn't uniform throughout. Does it matter?

Comment (tomjridge) [inline spec/13-syntax-summary.md:135]: The doc is informal, so arguably typos don't matter. There are already two quoting styles tho, and having a third just for this bracket seems excessive (what could be so special about _this_ bracket?)

Comment (som-snytt) [inline spec/13-syntax-summary.md:135]: They don't have a "shrug" emoji.

**Comment by SethTisue** [comment]
LGTM. thanks!

**Pull request merged**